packages/volto/src/config/index.js

```diff
@@ -8,6 +8,7 @@ import {
   layoutViewsNamesMapping,
 } from './Views';
 import { nonContentRoutes } from './NonContentRoutes';
+import { publicUiRoutes } from './PublicUiRoutes';
```

Comment:
Proposal: I would suggest a more specific name for this config key, like `publicNonContentRoutes`